functions/services/sendgrid/index.test.js

```diff
@@ -0,0 +1,57 @@
+const { addEmailToList } = require("./index");
+
+const faker = require("faker");
+
+jest.mock("@sendgrid/client");
+const client = require("@sendgrid/client");
+
+describe("SendgridService", () => {
+    beforeAll(() => {
+        process.env = {
+            SENDGRID_API_KEY: faker.random.uuid()
+        };
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe("addEmailToList", () => {
+        let fakeResponse = [
+            {
+                status: faker.random.number()
+            }
+        ];
+        beforeEach(() => {
+            client.request = jest.fn().mockResolvedValueOnce(fakeResponse);
+        });
+
+        it("adds the provided email to the list provided. Returns the response via a promise", async () => {
+            const fakeEmail = faker.internet.email();
+            const fakeListID = faker.random.number();
+
+            const response = await addEmailToList(fakeEmail, fakeListID);
```

Comment:
please add test here that the return value is resolved and not rejected

like that:
```js
const responsePr = addEmailToList(fakeEmail, fakeListID);
expect(responsePr).toResolve()
```

Reply:
Sure! I didn't know `toResolve` was an viable expect. I'll modify it, thanks for the feedback :)

Reply:
Updated. I didn't find `toResolve` to I resorted to using these instead:
```
expect(....).resolves.toBe(....)
```

Reply:
Yeah, it’s one of `jest-extended` function, in #38 I used it